client/stylesheets/_toCat.scss

```diff
@@ -4290,6 +4289,7 @@ On click the images will be displayed at normal size to complete the effect
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
+    margin: 2px 0 20px 0 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `margin` should be written more concisely as `2px 0 20px` instead of `2px 0 20px 0`